### PR TITLE
Improve live validation workflow

### PR DESCRIPTION
**Pull request by super-harsh**
**What this PR does / why we need it**:

- Increase wait for operator ready timeouts
- Omit kind tests from live validation workflow

### CODE REVIEW

File: .github/workflows/live-validation.yml [hunk 1]
```diff
@@ -28,7 +28,7 @@ jobs:
           echo "container_id=$container_id" >> $GITHUB_ENV
 
       - name: Run CI tasks against live resources
-        run: docker exec --env HOSTROOT=$GITHUB_WORKSPACE -e AZURE_TENANT_ID -e AZURE_CLIENT_ID -e AZURE_CLIENT_SECRET -e AZURE_SUBSCRIPTION_ID -e AZURE_CLIENT_SECRET_MULTITENANT -e AZURE_CLIENT_ID_MULTITENANT  -e AZURE_CLIENT_ID_CERT_AUTH -e AZURE_CLIENT_SECRET_CERT_AUTH "${{env.container_id }}" task ci-live
+        run: docker exec --env HOSTROOT=$GITHUB_WORKSPACE -e AZURE_TENANT_ID -e AZURE_CLIENT_ID -e AZURE_CLIENT_SECRET -e AZURE_SUBSCRIPTION_ID -e AZURE_CLIENT_SECRET_MULTITENANT -e AZURE_CLIENT_ID_MULTITENANT  -e AZURE_CLIENT_ID_CERT_AUTH -e AZURE_CLIENT_SECRET_CERT_AUTH "${{env.container_id }}" task ci-live --concurrency 2 # limit concurrency because build machines are small
```
Comment (theunrepentantgeek) [inline .github/workflows/live-validation.yml:31]: I didn't think they were small any longer.

**Review by theunrepentantgeek — APPROVED** [review]

**Comment by super-harsh** [comment]
Holding this PR off until we see a success in the live validation workflow.

File: .github/workflows/live-validation.yml [hunk 2]
```diff
@@ -1,14 +1,15 @@
 name: Live Azure Validation
 on:
   schedule:
-    # 13:00 UTC is a time when both US and NZ are asleep (or headed in that direction)
-    - cron: "0 13 * * *"
+    # 5:35 pm every Sunday (UTC)
+    # to ensure the docs are never too stale
```
Comment (theunrepentantgeek) [inline .github/workflows/live-validation.yml:5]: Update this comment.

File: v2/internal/controllers/crd_appconfiguration_configstore_test.go [hunk 3]
```diff
@@ -22,8 +22,10 @@ import (
 func Test_AppConfiguration_ConfigurationStore_CRUD(t *testing.T) {
 	t.Parallel()
 
-	createModeDefault := appconfig.ConfigurationStoreProperties_CreateMode_Default
-	publicNetworkAccessDisabled := appconfig.ConfigurationStoreProperties_PublicNetworkAccess_Disabled
+	if *isLive {
```
Comment (theunrepentantgeek) [inline v2/internal/controllers/crd_appconfiguration_configstore_test.go:25]: Can `isLive` ever be **nil**?

File: v2/internal/controllers/crd_dbforpostgresql_flexibleserver_20210601_test.go [hunk 4]
```diff
@@ -21,6 +21,10 @@ import (
 func Test_DBForPostgreSQL_FlexibleServer_20210601_CRUD(t *testing.T) {
 	t.Parallel()
 
+	if *isLive {
+		t.Skip("can't run in live mode, postresql flexible server takes too long to be provisioned and deletion")
```
Comment (theunrepentantgeek) [inline v2/internal/controllers/crd_dbforpostgresql_flexibleserver_20210601_test.go:25]:
```suggestion
		t.Skip("can't run in live mode, postresql flexible server takes too long to be provisioned and deleted")
```

File: v2/internal/controllers/crd_dbforpostgresql_flexibleserver_20220120preview_test.go [hunk 5]
```diff
@@ -22,6 +22,10 @@ import (
 func Test_DBForPostgreSQL_FlexibleServer_20220120preview_CRUD(t *testing.T) {
 	t.Parallel()
 
+	if *isLive {
+		t.Skip("can't run in live mode, postresql flexible server takes too long to be provisioned and deletion")
```
Comment (theunrepentantgeek) [inline v2/internal/controllers/crd_dbforpostgresql_flexibleserver_20220120preview_test.go:26]:
```suggestion
		t.Skip("can't run in live mode, postresql flexible server takes too long to be provisioned and deleted")
```

**Review by theunrepentantgeek — APPROVED** [review]
This is looking good. Interested to see what a matrix approach would look like too.

File: v2/internal/controllers/crd_appconfiguration_configstore_test.go (hunk 3 above)
Comment (super-harsh) [reply v2/internal/controllers/crd_appconfiguration_configstore_test.go:25]: The default would always be false

File: scripts/v2/wait-for-operator-ready.sh [hunk 6]
```diff
@@ -58,7 +58,7 @@ kubectl wait --for=condition=ready --timeout=3m pod -n azureserviceoperator-syst
 echo "Waiting for CRD cabundle..."
 export -f all_crds_have_cabundle
 export -f wait_for_crds_cabundle
-timeout 2m bash -c wait_for_crds_cabundle
+timeout 4m bash -c wait_for_crds_cabundle
```
Comment (matthchr) [inline scripts/v2/wait-for-operator-ready.sh:61]: This seems excessive to me. We shouldn't still be seeing this take that long on the beefy pool... is it?

**Review by matthchr — APPROVED** [review]

**Pull request merged**